tutorials/v7/ntuple/ntpl001_staff.C

```diff
@@ -15,9 +15,12 @@
 // Functionality, interface, and data format is still subject to changes.
 // Do not use for real data!
 
-// The following line should disappear in a future version of RNTuple, when
-// the common template specializations of RField are part of the LinkDef.h
+// Until C++ runtime modules are universally used, we explicitly load the ntuple library.  Otherwise the
+// use of templated types might prevent autoloading from taking place
```

Comment:
```suggestion
// Until C++ runtime modules are universally used, we explicitly load the ntuple library.  Otherwise
// triggering autoloading from the use of templated types would require an exhaustive enumeration
// of "all" template instances in the LinkDef file.
```